### PR TITLE
Fix release notes

### PR DESCRIPTION
**Pull request by leandro-lucarella-sociomantic**
The release notes were made before just another rename happened.

### CODE REVIEW

File: relnotes/version-alias.migration.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-* The `Version` alias to `versionInfo` provided by the `Version` module has been removed.
-  All remaining usage can be replaced with `versionInfo`.
+* Both `Version` and `versionInfo` aliases provided by the `Version` module has been removed.
```
Comment (nemanja-boric-sociomantic) [inline relnotes/version-alias.migration.md]: `have been removed`

**Comment by leandro-lucarella-sociomantic** [comment]
Updated and added another notes fixing commit.

File: relnotes/default-integration.migration.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 * Integration tests default location changed from `test` to `integrationtest`.
-  The location can be set explicitly via the variable `INTEGRATIONTEST`.
+
+  The location can be set explicitly via the variable `INTEGRATIONTEST`, but you are not required to define it anymore (as it was required in v1.x.x), you just need to define it if you don't want the new default (`integrationtest`).
```
Comment (mathias-lang-sociomantic) [inline relnotes/default-integration.migration.md:3]: Sometimes I'm wondering if release notes should address the user directly, or use a more passive tense, e.g. "it is not required to define it...".

**Review by mathias-lang-sociomantic — APPROVED** [review]
LGTM

File: relnotes/default-integration.migration.md (hunk 2 above)
Comment (leandro-lucarella-sociomantic) [reply relnotes/default-integration.migration.md:3]: The never ending dilemma when writing documentation in general...

**Review by nemanja-boric-sociomantic — APPROVED** [review]

**Pull request merged**